tests/integration/golang/aim/metric/search_test.go

```diff
@@ -315,14 +316,14 @@ func (s *SearchMetricsTestSuite) Test_Ok() {
 		{
 			name: "SearchMetric",
 			request: request.SearchMetricsRequest{
-				Metrics: []request.MetricTuple{
+				Metrics: [][]string{
 					{
-						Key:     "TestMetric1",
-						Context: "{}",
+						"TestMetric1",
```

Comment:
This change makes me think the old `Key` and `Context` were clearer than `[]string` for describing a metric -- is it worth retaining that JSON format when the request is generated in the UI?